CONTRIBUTING.md

```diff
@@ -74,10 +74,15 @@ Run a single test
 `$ rake test TEST=path/to/test.rb TESTOPTS="--name=test_something"`
 
 Run tests against different Rails versions by setting the RAILS_VERSION variable
-and bundling gems.
+and bundling gems.  (save this script somewhere executable and run from top of AMS repository)
 
 ```bash
-for version in 4.0 4.1 4.2 master; do
+#!/usr/bin/env bash
+
+rcommand='puts YAML.load_file("./.travis.yml")["env"]["matrix"].join(" ").gsub("RAILS_VERSION=", "")'
+versions=$(ruby -ryaml -e "$rcommand")
+
+for version in ${versions[@]}; do
```

Comment:
@cgmckeever I am envious of your bash skills.  I've never seen that and it is awesome